internal/commands/database/create_test.go

```diff
@@ -0,0 +1,173 @@
+package database
+
+import (
+	"testing"
+
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/commands"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/commands/storage"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/config"
+	smock "github.com/UpCloudLtd/upcloud-cli/v3/internal/mock"
+	"github.com/UpCloudLtd/upcloud-cli/v3/internal/mockexecute"
+	"github.com/UpCloudLtd/upcloud-go-api/v8/upcloud"
+	"github.com/UpCloudLtd/upcloud-go-api/v8/upcloud/request"
+	"github.com/stretchr/testify/assert"
+)
+
+var (
+	Title1             = "mock-storage-title1"
+	Title2             = "mock-storage-title2"
+	UUID1              = "0127dfd6-3884-4079-a948-3a8881df1a7a"
+	UUID3              = "012c61a6-b8f0-48c2-a63a-b4bf7d26a655"
+	PrivateNetworkUUID = "03b5b0a0-ad4c-4817-9632-dafdb3ace5d9"
+	MockPrivateIPv4    = "10.0.0.1"
+	MockPrivateIPv6    = "fd42:42::1"
+	MockPublicIPv4     = "192.0.2.0"
+	MockPublicIPv6     = "2001:DB8::1"
+)
+
+func TestCreateCommand(t *testing.T) {
+	databaseDetailsMaint := upcloud.ManagedDatabase{
+		UUID: UUID1,
```

Comment:
Seems that only `UUID1` is used, so other vars could be removed from here. Maybe even inline `UUID1` since it is only used once 🤔 

```suggestion
func TestCreateCommand(t *testing.T) {
	databaseDetailsMaint := upcloud.ManagedDatabase{
		UUID: "0927dfd6-3884-4079-a948-3a8881df1a7a",
```